Formula/xcb-proto.rb

```diff
@@ -1,10 +1,18 @@
+class SystemPython3Requirement < Requirement
+  fatal true
+  satisfy { which "python3" }
+  def message
+    "An existing non-Homebrew Python 3 installation is required in order to avoid cyclic dependencies!"
```

Comment:
```suggestion
    "An existing Python 3 installation is required in order to avoid cyclic dependencies (as Homebrew's Python depends on xcb-proto)."
```